packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -384,12 +385,25 @@ pub trait EntityStore: for<'q> crud::Read<PersistedEntity, Query<'q> = Filter<'q
     /// - if the [`Entity`] doesn't exist
     /// - if the [`EntityType`] doesn't exist
     /// - if the [`Entity`] is not valid with respect to its [`EntityType`]
-    /// - if the account referred to by `updated_by` does not exist
+    /// - if the account referred to by `actor_id` does not exist
     async fn update_entity(
         &mut self,
         entity_id: EntityId,
         entity: Entity,
         entity_type_id: VersionedUri,
         actor_id: UpdatedById,
     ) -> Result<PersistedEntityMetadata, UpdateError>;
+
+    /// Archives an [`Entity`].
+    ///
+    /// # Errors:
+    ///
+    /// - if the [`Entity`] doesn't exist
+    /// - if the [`Entity`] has already been archived
+    /// - if the account referred to by `actor_id` does not exist
```

Comment:
We won't be able to identify between the first and the second without going out of our way to do so. But yes, it's used as a context switch using the other methods as a guide for how it's done for this new method.

Reply:
Then we could perhaps say 
```suggestion
    /// - if there isn't an [`Entity`] associated with the `entityId` in the latest entities table
    ///   - this could be because the [`Entity`] doesn't exist, or
    ///   - the [`Entity`] has already been archived
    /// - if the account referred to by `actor_id` does not exist
```

But I'm just nitpicking (caveat, not sure my suggestion is rustfmt happy, and maybe the `entityId` thing could be a doc link)

Reply:
Addressed in 2926334f19c75a6d1d626e8b1a71522715566ec4